_opcodes/op233.html

```diff
@@ -13,7 +13,7 @@
 Sets the weapon proficiency points for the weapon type specified in the 'Type' field, to the value specified by the 'Amount' field, for the targeted creature(s).
 
 {% capture note %}
-The Proficiency Modifier effect cannot increment or decrement proficiency points, it can only SET them.
+The Proficiency Modifier effect cannot increment or decrement proficiency points, it can only SET them - This is not true for EEs. Indeed, param#2 ('Type') is split into two subfields. Via setting the second subfield to 1 (INCREMENT), it's possible to increment/decrement proficiency points.
```

Comment:
So how do you decrement them? How is the subfield being split and where ...

Reply:
> So how do you decrement them? How is the subfield being split and where ...

Certain opcodes (e.g., opcode #12, 'damage' or opcode #233) split the `parameter2` field into two subfields: each is a SHORT, located respectively at 0x8 ﻿and 0xa (SPL and ITM).

As you probably know, Rangers are granted a bonus of 2 to Two-Weapon Style. You can nullify it via something like this:
```
LPF ADD_SPELL_EFFECT
INT_VAR
  opcode = 233  // Modify proficiency
  target = 2
  parameter1 = "-2"
  parameter2 = 114 + (1 << 16)  // Two Weapon Skill, Increment
  timing = 9  // Instant/Permanent
END
```

Reply:
Ok, so the high bits of param2 are mode bits (with only one user). This is what needs to be added in to be clear.